### PR TITLE
Add Achievement: Bluffy the Slayer (kill 500 of any giant bat); stub for unicorn horn quest (NPC, Map, item pegasus feather)

### PR DESCRIPTION
**Pull request by tobiwankenobi-stendhal**

### CODE REVIEW

File: src/games/stendhal/server/maps/quests/UnicornHornsForRer.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
+package games.stendhal.server.maps.quests;
+import games.stendhal.common.parser.Sentence;
+import games.stendhal.server.entity.npc.*;
+import games.stendhal.server.entity.npc.action.*;
+import games.stendhal.server.entity.npc.condition.*;
+import java.util.*;
+
```
Comment (AntumDeluge) [inline src/games/stendhal/server/maps/quests/UnicornHornsForRer.java]: These imports aren't used & should be deleted:
- `games.stendhal.common.parser.Sentence`
- `games.stendhal.server.entity.npc.condition.*`
- `java.util.*`

File: src/games/stendhal/server/maps/quests/UnicornHornsForRer.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+	public String getSlotName() {
+		return QUEST_SLOT;
+	}
+
+	@Override
+	public void addToWorld() {
+		fillQuestInfo(
+				"Unicorn Horns for Rer",
+				"Rer Ecros promised to make me stronger if I bring him "+REQUIRED_HORNS+" unicorn horns and "
+						+ (REQUIRED_FEATHERS > 1? REQUIRED_FEATHERS + " pegasus feathers.":"one pegasus feather."),
```
Comment (AntumDeluge) [inline src/games/stendhal/server/maps/quests/UnicornHornsForRer.java]: Dead code here.

File: src/games/stendhal/server/maps/quests/UnicornHornsForRer.java (hunk 1 above)
Comment (AntumDeluge) [reply src/games/stendhal/server/maps/quests/UnicornHornsForRer.java]: If you are using [Eclipse](https://eclipse.org/), just hit `Ctrl+Shift+O`.

**Comment by tobiwankenobi-stendhal** [comment]
Made the required changes, 
added copyright-info for pegasus_feather.png,
disabled "reset"-option in unicorn-horns-quest

**Pull request closed without merging**